indra/newview/llluamanager.h

```diff
@@ -84,6 +84,9 @@ class LLLUAmanager
 
     static const std::map<std::string, std::string> getScriptNames() { return sScriptNames; }
 
+    static S32 sAutorunScriptCount;
+    static S32 sScriptCount;
```

Comment:
Maybe a stats struct? I can think of a few additional Lua-related statistics of potential interest, even in the short term:
* the high-water mark for number of concurrently running Lua scripts (Lua-targeted C++ coroutines)
* stats based on totaling interrupts: low, median, high interrupt count per script; total Lua interrupts for the viewer session.

Lua interrupts don't correlate with real time, or even with Lua bytecode steps, but they're sort of like gauges measuring "engine hours" for heavy equipment: they indicate (in an admittedly unspecified way) the number of cycles the Lua engine spends doing script work. We might capture "kilointerrupts" if the numbers get absurdly large.

Reply:
Yep, it makes sense to use a struct if we have more than 2-3 stats.
Agree, those two may be useful. We should discuss it.
 But I can imagine that a script with an infinite loop can spoil total Lua interrupts for the viewer session.

Reply:
Okay, how about: we count interrupts for individual scripts, deferring contributing to the viewer session total until script termination. But if we terminate a script with the "infinite loop" error, we zero its interrupt statistic so it doesn't throw off stats.

Reply:
Yes, it makes sense.